src/components/HeaderBreadcrumbs/HeaderBreadcrumbsItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {ReactNode, useMemo} from 'react';
```

Comment:
Why do we add new component in pr with analytics are they related with each other?

Reply:
I had looked carefully on this code and figured out - there is overcoding) we do not need to create new Components with analytics handlers in the PR. At least not now